todo_app/app.py

```diff
@@ -79,8 +79,8 @@ def something():
         return redirect(url_for('get_cards'))
 
     
-    @writer_required
     @app.route('/', methods = ['POST'])
+    @writer_required
```

Comment:
I can see you also made this change to the `complete_card` method but then reverted it, which broke that methods authorization!